src/actions/actions/linode-boot/__init__.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+######################################################################
+# Cloud Routes Bridge
+# -------------------------------------------------------------------
+# Actions Module
+######################################################################
+
+import requests
+import json
+import time
+
+
+def action(**kwargs):
+    ''' This method is called to action a reaction '''
+    redata = kwargs['redata']
+    jdata = kwargs['jdata']
+    logger = kwargs['logger']
+    run = True
+    # Check for Trigger
+    if redata['trigger'] > jdata['failcount']:
+        run = False
+
+    # Check for lastrun
+    checktime = time.time() - float(redata['lastrun'])
+    if checktime < redata['frequency']:
+        run = False
+
+    if redata['data']['call_on'] not in jdata['check']['status']:
+        run = False
+
+    if run:
+        return call_linode(redata, jdata, logger)
+    else:
+        return None
+
+
+def call_linode(redata, jdata, logger):
+    ''' Perform actual call '''
+    url = 'https://api.linode.com/'
+    params = {
+        "api_action": "linode.reboot",
+        "LinodeID": int(redata['data']['linode_id']),
+        "api_key": str(redata['data']['api_key'])
+    }
+    payload = json.dumps(params)
```

Comment:
Looks like Linodes API expects them to be POST params vs a JSON message.

> Request Format
> The api_action parameter is required on every request. All API request parameters and method parameters can be sent via either GET or POST variables.
>
> Example request:

https://api.linode.com/?api_key=sekrit&api_action=test.echo&foo=bar
NOTE: api.linode.com responds only via https://api.linode.com.